build.rs

```diff
@@ -19,4 +19,7 @@ fn main() {
     if ac.probe_rustc_version(1, 51) {
         println!("cargo:rustc-cfg=raw_ref_macros");
     }
+    if ac.probe_rustc_version(1, 65) {
```

Comment:
We should then probably also have 1.65.0 listed in https://github.com/Gilnaa/memoffset/blob/master/.github/workflows/ci.yml ?